python/lsst/summit/utils/assessPeekExposure.py

```diff
@@ -0,0 +1,200 @@
+from astropy.table import Table
+from astropy.utils.exceptions import AstropyUserWarning
+from lsst.afw.geom import ellipses
+from lsst.daf.butler.datastore.cache_manager import DatastoreCacheManager
+from lsst.summit.utils.bestEffort import BestEffortIsr
+from lsst.summit.utils.peekExposure import PeekExposureTask
+from tqdm import tqdm
+
+# Set logger level to higher than CRITICAL to suppress all output
+silent_logger = logging.getLogger("silent_logger")
+silent_logger.setLevel(logging.CRITICAL + 1)
+
+
+# get one BestEffortIsr,afwDisplay per subprocess
+def initializer():
+    global bestEffort, pet, display, fig, ax
+    fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(9, 9), facecolor="k")
+    display = afwDisplay.Display(fig, backend="matplotlib")
+    display.scale("asinh", -1, 31)
+    petConfig = PeekExposureTask.ConfigClass()
+    pet = PeekExposureTask(config=petConfig, log=silent_logger, display=display)
+    bestEffort = BestEffortIsr(embargo=True)
+
+
+# retrieve best-effort-isr and run PET on it
+def doit(idx, row, doPlot):
+    exposureId = row["exposureId"]
+    dataId = {"instrument": "LATISS", "exposure": exposureId, "detector": 0}
+    exp = bestEffort.getExposure(dataId)
+    mode = "auto"
+    donutDiameter = None
+
+    # Incorrectly labeled as photo
+    if exposureId >= 2022101200584 and exposureId <= 2022101200588:
+        mode = "donut"
+        donutDiameter = 1500
+
+    # Incorrectly labeled as giant donut
+    if exposureId >= 2022101200589 and exposureId <= 2022101201247:
+        mode = "photo"
+
+    t0 = time.time()
+    result = pet.run(exp, mode=mode, donutDiameter=donutDiameter, doDisplay=doPlot)
+    t1 = time.time()
+    dt = t1 - t0
+
+    ax.text(
+        0.01,
+        0.97,
+        f"Runtime: {dt:.2f} s",
+        c="w",
+        transform=ax.transAxes,
+        fontsize=12,
+    )
+
+    inTag = row["finalTag"]
+    true_x = row["centroid_x"]
+    true_y = row["centroid_y"]
+    found_x = result.brightestCentroid.x
+    found_y = result.brightestCentroid.y
+
+    if np.isfinite(true_x):
+        n_x = (true_x - (result.binSize - 1) / 2) / result.binSize
+        n_y = (true_y - (result.binSize - 1) / 2) / result.binSize
+        if doPlot:
+            display.dot("o", n_x, n_y, ctype=afwDisplay.MAGENTA, size=20)
+
+        if np.isfinite(found_x):
+            dist = np.sqrt((true_x - found_x) ** 2 + (true_y - found_y) ** 2)
+            if dist < 20:
+                outTag = "<2"
+            elif dist < 100:
+                outTag = "<10"
+            else:
+                outTag = ">10"
+        else:
+            outTag = "noStar"
+    else:
+        if np.isfinite(found_x):
+            outTag = "noTruth"
+        else:
+            outTag = "nothing"
+
+    if doPlot:
+        if np.isfinite(result.psfShape.getIxx()):
+            distortion = ellipses.SeparableDistortionDeterminantRadius(result.psfShape)
+            fwhm = 2.35 * distortion.getDeterminantRadius() * 0.1
```

Comment:
I think I can guess what these random numbers mean, but it'd be better to make them module- or class-variable named constants.